frontend/packages/console-dynamic-plugin-sdk/src/extensions/import-environments.ts

```diff
@@ -0,0 +1,30 @@
+import { Extension, ExtensionDeclaration } from '../types';
+
+type ImageEnvironment = {
+  /** Environment variable key */
+  key: string;
+  /** The input field's label */
+  label: string;
+  /** Default value to use as a placeholder */
+  default?: string;
```

Comment:
My first thought was this should be named `value` or `defaultValue`.

Reply:
Updated to `defaultValue`.